spec/acceptance/updating_posix_acl_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
```

Comment:
This now overlaps with the other test. I don't know if that works well.

Reply:
ah. I can move it to the existing file.

Reply:
The alternative is to use a new name, but either works.